### PR TITLE
Kaamil/feature/11

### PR DESCRIPTION
**Pull request by kslokhandwala2022**
Created some functionality and added some colors to my( Kaamil's) personal UI page

### CODE REVIEW

**Review by Kohmei358 — APPROVED** [review]

File: src/main/java/edu/wpi/teamname/App.java [hunk 1]
```diff
@@ -17,7 +17,7 @@ public void init() {
 
   @Override
   public void start(Stage primaryStage) throws Exception {
-    Parent root = FXMLLoader.load(getClass().getResource("views/MainPage.fxml"));
+    Parent root = FXMLLoader.load(getClass().getResource("views/Kohmei.fxml"));
```
Comment (Kohmei358) [inline src/main/java/edu/wpi/teamname/App.java:20]: We should switch this back

File: src/main/java/edu/wpi/teamname/views/MainPage.java [hunk 2]
```diff
@@ -15,7 +18,30 @@
...
+  private String cssSheet =
+      "file:/C:/Users/Kaam9/Documents/WPI/CS/Soft/CS3733-C21-Team-U-Project-BWApp/build/resources/main/edu/wpi/teamname/views/styles/Kaamil2.css";
+
+  @FXML
+  private void updateMessage() {
+    msgKaamil.setText(inputKaamil.getText());
+  }
+
+  @FXML
+  private void toggleColor() {
```
Comment (Kohmei358) [inline src/main/java/edu/wpi/teamname/views/MainPage.java:39]: Cool funciton

**Review by Kohmei358 — CHANGES_REQUESTED** [review]
Default page should be main

**Review by Kohmei358 — APPROVED** [review]

**Review by tasanderv — APPROVED** [review]

**Comment by tasanderv** [comment]
full send (combining kaamil and main)

**Pull request merged**